.circleci/rahti-ci.sh

```diff
@@ -18,13 +18,21 @@ else
 fi
 
 docker build --pull \
+    --tag rems:latest \
+    --tag rems:${tag2} \
     --tag docker-registry.rahti.csc.fi/rems/rems:${tag1} \
     --tag docker-registry.rahti.csc.fi/rems/rems:${tag2} .
 
 docker login -p $rahtitoken -u unused docker-registry.rahti.csc.fi
 docker push docker-registry.rahti.csc.fi/rems/rems:${tag1}
 docker push docker-registry.rahti.csc.fi/rems/rems:${tag2}
 
+if [ "${tag1}" == "release" ] ; then
+    docker login -u remspush -p ${dockerhub}
+    docker push rems:latest
+    docker push rems:${tag2}
+fi
```

Comment:
how about my idea of splitting the docker build & push out of the rahti deploy?

Reply:
Done